src/Rules/Deprecations/DeprecationHelper.php

```diff
@@ -0,0 +1,57 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Deprecations;
+
+use PHPStan\Broker\Broker;
+use PHPStan\Reflection\ClassReflection;
+
+trait DeprecationHelper
```

Comment:
I really dislike traits. Make this a service that's injected into the constructor instead. Thank you.